packaging/rhel/xrootd.spec.in

```diff
@@ -733,6 +733,10 @@ fi
 %{_libdir}/libXrdN2No2p-4.so
 %{_libdir}/libXrdOssSIgpfsT-4.so
 %{_libdir}/libXrdServer.so.*
+%{_libdir}/libXrdSsi-4.so.*
+%{_libdir}/libXrdSsiLib.so.*
+%{_libdir}/libXrdSsiLog-4.so.*
```

Comment:
Same here, since libXrdSsiLog is not versioned this fails as there are no libXrdSsiLog-4.si.*, there is only the libXrdSsiLog-4.so, which in turn is installed but not packaged.